lib/KBaseSearchEngine/KBaseSearchEngineClient.pm

```diff
@@ -760,7 +760,7 @@ Retrieve objects by their GUIDs.
 {
     my($self, @args) = @_;
 
-# Authentication: required
+# Authentication: requred
```

Comment:
This is misspelled, which is odd since it's a generated file. Did you edit the generated files manually? If so just recompile